### PR TITLE
hw10.2

### PR DESCRIPTION
**Pull request by Mykhaylo12**

### CODE REVIEW

File: src/main/java/core/basesyntax/SymmetricDifference.java [hunk 1]
```diff
@@ -10,6 +11,18 @@
  */
 public class SymmetricDifference<T> {
     public Set<T> symmetricDifference(Set<? extends T> set1, Set<? extends T> set2) {
-        return null;
+        Set<T> symmetricDifference = new TreeSet<>();
+        for (T valueFromSet1 : set1) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/SymmetricDifference.java]: good, but let's using standard methods `removeAll` or `retainAll` and `addAll`

File: src/main/java/core/basesyntax/SymmetricDifference.java [hunk 2]
```diff
@@ -10,6 +11,18 @@
...
+            if (!set2.contains(valueFromSet1)) {
+                symmetricDifference.add(valueFromSet1);
+            }
+        }
+        for (T valueFromSet2 : set2) {
+            if (!set1.contains(valueFromSet2)) {
+                symmetricDifference.add(valueFromSet2);
+            }
+        }
+        System.out.println(symmetricDifference);
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/SymmetricDifference.java]: do we need sout here?

Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/SymmetricDifference.java]: forgot to remove after testing

File: src/main/java/core/basesyntax/SymmetricDifference.java (hunk 1 above)
Comment (Mykhaylo12) [reply src/main/java/core/basesyntax/SymmetricDifference.java]: ok

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**